nerfstudio/models/splatfacto.py

```diff
@@ -757,16 +783,27 @@ def get_outputs(self, camera: Cameras) -> Dict[str, Union[torch.Tensor, List]]:
             info["means2d"].retain_grad()
         self.xys = info["means2d"]  # [1, N, 2]
         self.radii = info["radii"][0]  # [N]
-
         alpha = alpha[:, ...]
+
+        background = self._get_background_color()
         rgb = render[:, ..., :3] + (1 - alpha) * background
         rgb = torch.clamp(rgb, 0.0, 1.0)
+
         if render_mode == "RGB+ED":
             depth_im = render[:, ..., 3:4]
             depth_im = torch.where(alpha > 0, depth_im, depth_im.detach().max()).squeeze(0)
         else:
             depth_im = None
-        return {"rgb": rgb.squeeze(0), "depth": depth_im, "accumulation": alpha.squeeze(0), "background": background}  # type: ignore
+
+        if background.shape[0] == 3:
+            background = background.expand(H, W, 3)
```

Comment:
Why do you want to expand background dims?

Reply:
Becaue the viser will broke if tries to check the background without expanding it.